creatio/theme.json

```diff
@@ -0,0 +1,728 @@
+{
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"background": true,
+			"custom": true,
+			"customDuotone": true,
+			"customGradient": true,
+			"defaultDuotone": true,
+			"defaultGradients": true,
+			"defaultPalette": true,
+			"duotone": [],
+			"gradients": [],
+			"palette": [
+				{
+					"color": "#11181F",
+					"name": "Primary",
+					"slug": "primary"
+				},
+				{
+					"color": "#005ae1",
+					"name": "Secondary",
+					"slug": "secondary"
+				},
+				{
+					"color": "#474747",
+					"name": "Foreground",
+					"slug": "foreground"
+				},
+				{
+					"color": "#ffffff",
+					"name": "Background",
+					"slug": "background"
+				},
+				{
+					"color": "#f7f9fd",
+					"name": "Tertiary",
+					"slug": "tertiary"
+				}
+			],
+			"text": true
+		},
+		"layout": {
+			"contentSize": "620px",
+			"definitions": {
```

Comment:
I think these "definitions" are added by Gutenberg right now (and shouldn't be).  I think they should be removed.  


... or maybe I just don't understand the purpose because my brain is out of date?